Tests/M5Core2TestApp/Program.cs

```diff
@@ -1,94 +1,110 @@
 
     Debug.WriteLine($"Touch Panel Event Received Category= {e.EventCategory} Subcategory= {e.TouchEventCategory}");
     Console.CursorLeft = 0;
     Console.CursorTop = 0;
 
-    Debug.WriteLine(StrXY1 + e.X + StrXY2 + e.Y + StrID + e.Id);
-    Console.WriteLine(StrXY1 + e.X + StrXY2 + e.Y + StrID + e.Id + "  ");
+    var eventMsg = $"{StrTouchEvent}({e.X},{e.Y}){StrEventID}{e.Id}";
+    Debug.WriteLine(eventMsg);
+    Console.WriteLine(eventMsg);
 
     if ((e.TouchEventCategory & TouchEventCategory.LeftButton) == TouchEventCategory.LeftButton)
     {
+        Vibrate(150);
         Debug.WriteLine(StrLB);
         Console.WriteLine(StrLB);
     }
     else if ((e.TouchEventCategory & TouchEventCategory.MiddleButton) == TouchEventCategory.MiddleButton)
     {
+        Vibrate(150);
         Debug.WriteLine(StrMB);
         Console.WriteLine(StrMB);
     }
     else if ((e.TouchEventCategory & TouchEventCategory.RightButton) == TouchEventCategory.RightButton)
     {
+        Vibrate(150);
         Debug.WriteLine(StrRB);
         Console.WriteLine(StrRB);
     }
 
     if ((e.TouchEventCategory & TouchEventCategory.Moving) == TouchEventCategory.Moving)
     {
-        Debug.WriteLine(StrMove);
-        Console.Write(StrMove);
+        Debug.WriteLine($"{StrTouchEventCatagory}{StrMove}");
+        Console.WriteLine($"{StrTouchEventCatagory}{StrMove}");
     }
 
     if ((e.TouchEventCategory & TouchEventCategory.LiftUp) == TouchEventCategory.LiftUp)
     {
-        Debug.WriteLine(StrLiftUp);
-        Console.Write(StrLiftUp);
+        Debug.WriteLine($"{StrTouchEventCatagory}{StrLiftUp}");
+        Console.WriteLine($"{StrTouchEventCatagory}{StrLiftUp}");
     }
 
-    if ((e.TouchEventCategory & TouchEventCategory.DoubleTouch) == TouchEventCategory.DoubleTouch)
+    if ((e.TouchEventCategory & TouchEventCategory.DoubleTouch) == TouchEventCategory.DoubleTouch) //TODO: never seems to fire!
     {
-        Debug.WriteLine(StrDoubleTouch);
-        Console.Write(StrDoubleTouch);
+        Debug.WriteLine($"{StrTouchEventCatagory}{StrDoubleTouch}");
+        Console.WriteLine($"{StrTouchEventCatagory}{StrDoubleTouch}");
     }
 
-    Console.WriteLine("                                    ");
-    Console.WriteLine("                                    ");
-    Console.WriteLine("                                    ");
+    Console.WriteLine("");
+    Console.WriteLine("");
+
+    Console.WriteLine($"{StrGyro}:");
+    Console.WriteLine($"  ({mpu6886.GetGyroscope().X:N2},{mpu6886.GetGyroscope().Y:N2},{mpu6886.GetGyroscope().Z:N2})");
+
+    Console.WriteLine($"{StrAcc}:");
+    Console.WriteLine($"  ({mpu6886.GetAccelerometer().X:N2},{mpu6886.GetAccelerometer().Y:N2},{mpu6886.GetAccelerometer().Z:N2})");
+    
+    Debug.WriteLine($"{StrCpuT}{M5Core2.Power.GetInternalTemperature().DegreesCelsius}�C");
+    Console.WriteLine($"{StrCpuT}{M5Core2.Power.GetInternalTemperature().DegreesCelsius}_C"); //TODO: note that we cannot use the proper symbol!
```

Comment:
OK for adding this warning note, of course. Please follow the code style an move it to the line before, not in the same code line.

Reply:
Note that the symbol can be added in the font. It's just not been extracted. But right, better to write it!

Reply:
Improved the comment.